src/Wfirma/Invoice/WfirmaInvoiceItem.php

```diff
@@ -32,8 +32,12 @@ public function print(Media $media): Media
         $content->with('price', $this->price->toString());
 
         if ($this->vatId) {
-            $vatCode = $content->with('vat_code', '');
-            $vatCode->with('id', $this->vatId);
+            if (WfirmaValueAddedTax::NO_VAT === $this->vatId) {
```

Comment:
jak nie chcesz zmienic tego tax rate na stringa tylko to robić w ten sposób, to może już niech to pójdzie lepiej jako

if (WfirmaValueAddedTax::NO_VAT === $this->vatId) {
}
else if ($this->vatId) 
{
}
else {
}